doc/source/releases/15.05.rst

```diff
@@ -0,0 +1,348 @@
+.. to_doc
+
+-------------------------------
+15.05
+-------------------------------
+
+
+Enhancements
+-------------------------------
+
+* Pluggable framework to custom authentication (including new LDAP/Active
+  Directory integration). Thanks to many including Andrew Robinson,
+  Nicola Soranzo, and David Trudgian. `Pull Request 1`_, `Pull Request 33`_,
+  `Pull Request 51`_, `Pull Request 74`_, `Pull Request 98`_,
+  `Pull Request 216`_
+* Implement a new ``section`` tag for tools. `Pull Request 35`_,
```

Comment:
s/tools/tool parameters/

Otherwise it may seem you're speaking of the tool menu in the left pane.